clang/lib/StaticAnalyzer/Core/MemRegion.cpp

```diff
@@ -1022,6 +1022,23 @@ getStackOrCaptureRegionForDeclContext(const LocationContext *LC,
   return (const StackFrameContext *)nullptr;
 }
 
+static bool isStdStreamVar(const VarDecl *D) {
+  const auto *ND = dyn_cast<NamedDecl>(D);
+  if (!ND)
+    return false;
+  DeclarationName DeclN = ND->getDeclName();
+  if (!DeclN.isIdentifier())
+    return false;
```

Comment:
I think a NamedDecl has the memfn `getIdentifier` which either returns the II or NULL.
Consequently, this code could be further simplified.